newdle/client/src/components/creation/CreationHeader.js

```diff
@@ -6,6 +6,7 @@ import {useDispatch, useSelector} from 'react-redux';
 import {getStep, getFullTimeslots} from '../../selectors';
 import {setStep} from '../../actions';
 import styles from './CreationHeader.module.scss';
+import {STEPS} from './steps';
 
 export default function CreationHeader() {
   const steps = [
```

Comment:
maybe this should go into `steps.js` (and be turned into a dict using the steps you defined there as an index)?

Reply:
Had the same idea before but then realized only `CreationHeader` needs this information.
Nonetheless, I was still considering to move it until I noticed that making the dict keys being step strings such as `STEPS.PARTICIPANTS` but with the values being an object with `title` and `description` causes us to not being able to use them like numbers (like it happens in `CreationHeader` just below):
```javascript
setStep(STEPS.PARTICIPANTS + 1)
```
If this still seems useful, I can change it.